protostar/commands/deploy_command.py

```diff
@@ -39,19 +39,26 @@ def description(self) -> str:
 
     @property
     def example(self) -> Optional[str]:
-        return "protostar deploy ./build/main.json --network testnet"
+        return "protostar deploy 0x4221deadbeef123 --network testnet"
 
     @property
     def arguments(self):
         return [
             BLOCK_EXPLORER_ARG,
             ProtostarArgument(
-                name="contract",
-                description="The path to the compiled contract.",
-                type="path",
-                is_required=True,
+                name="class_hash",
+                description="The hash of the declared contract class.",
+                type="class_hash",
                 is_positional=True,
             ),
+            ProtostarArgument(
+                name="max-fee",
+                description=(
+                    "The maximum fee that the sender is willing to pay for the transaction. "
+                    'Provide "auto" to auto estimate the fee.'
+                ),
+                type="fee",
+            ),
```

Comment:
perhaps we could extract this into commons module?

Reply:
Out of scope of this PR